src/pudl/metadata/fields.py

```diff
@@ -2841,12 +3512,16 @@
         "type": "string",
         "description": "Utility name, chosen arbitrarily from the several possible utility names available in the utility matching process. Included for human readability only.",
     },
-    "utility_owned_capacity_mw": {"type": "number", "unit": "MW"},
+    "utility_owned_capacity_mw": {
+        "type": "number",
+        "description": "Total non-net-metered capacity owned by the respondent.",
+        "unit": "MW",
+        # TODO: Make this column name more specific to non-net metered capacity.
+    },
```

Comment:
This column comes from the `core_eia861__yearly_non_net_metering_misc` table, maybe we should clarify in the column name that it's from non-net_metered capacity?

Reply:
Added to #3301